docs/codepush.md

```diff
@@ -0,0 +1,122 @@
+import { DatadogCodepush } from '@datadog/mobile-react-native-code-push';
+
+const config = new DdSdkReactNativeConfiguration(
+    '<CLIENT_TOKEN>',
+    '<ENVIRONMENT_NAME>',
+    '<RUM_APPLICATION_ID>',
+    true, // track User interactions (e.g.: Tap on buttons. You can use 'accessibilityLabel' element property to give tap action the name, otherwise element type will be reported)
+    true, // track XHR Resources
+    true // track Errors
+);
+
+await DatadogCodepush.initialize(config);
+```
+
+## Upload CodePush source maps
+
+Install [`@datadog/datadog-ci`][3] as a development dependency to your project.
+
+To install it with npm:
+
+```sh
+npm install @datadog/datadog-ci --save-dev
+```
+
+To install it with yarn:
+
+```sh
+yarn add -D @datadog/datadog-ci
+```
+
+Create a gitignored `datadog-ci.json` file at the root of your project containing your API key and the Datadog site (if not `datadoghq.com`):
+
+```json
+{
+    "apiKey": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+    "site": "datadoghq.eu"
+}
+```
+
+**Note:** You can also export them as `DATADOG_API_KEY` and `DATADOG_SITE` environment variables.
+
+When releasing a new CodePush bundle, specify a directory for outputting the source maps and bundle:
+
+```sh
+appcenter codepush release-react -a MyOrganization/MyApplication -d MyDeployment --sourcemap-output --output-dir ./build
+```
+
+Then run the `datadog-ci react-native codepush` command by passing the adequate CodePush `app` and `deployment` arguments.
+
+To run it with npm:
+
+```sh
+npm run datadog-ci react-native codepush --platform ios --service com.company.app --bundle ./build/CodePush/main.jsbundle --sourcemap ./build/CodePush/main.jsbundle.map --app MyOrganization/MyApplication --deployment MyDeployment
+```
+
+To run it with yarn:
+
+```sh
+yarn datadog-ci react-native codepush --platform ios --service com.company.app --bundle ./build/CodePush/main.jsbundle --sourcemap ./build/CodePush/main.jsbundle.map --app MyOrganization/MyApplication --deployment MyDeployment
+```
+
+## Alternatives
+
+These steps will ensure that the `version` will match the format `{commercialVersion}-codepush.{codePushLabel}`, such as `1.2.4-codepush.v3`.
+
+You can also do that by specifying a `versionSuffix` in the SDK configuration:
+
+```js
+const config = new DdSdkReactNativeConfiguration(
+    '<CLIENT_TOKEN>',
+    '<ENVIRONMENT_NAME>',
+    '<RUM_APPLICATION_ID>',
+    true, // track User interactions (e.g.: Tap on buttons. You can use 'accessibilityLabel' element property to give tap action the name, otherwise element type will be reported)
+    true, // track XHR Resources
+    true // track Errors
+);
+
+config.versionSuffix = `codepush.${codepushVersion}`; // will result in "1.0.0-codepush.v2"
+```
+
+Be aware that to avoid potential version clashes, `versionSuffix` adds a dash (`-`) before the suffix.
+You can obtain the `codepushVersion` by hardcoding it or using [`CodePush.getUpdateMetadata`][4].
```

Comment:
```suggestion
In order to avoid potential version clashes, the `versionSuffix` adds a dash (`-`) before the suffix.

To obtain the `codepushVersion`, you can hardcode it or use [`CodePush.getUpdateMetadata`][4].
```